Documentation/coding-guidelines/breaking-change-rules.md

```diff
@@ -158,16 +158,17 @@ Breaking Change Rules
 * Adding `virtual` to a member  
 > Make note, that marking a member virtual might cause previous consumers to still call the member non-virtually.
 
-* Introducing or removing an override
-> Make note, that introducing an override might cause previous consumers to skip over the override when calling `base`.
-
 &#10007; **Disallowed**
 * Adding an member to an interface
 
 * Adding an abstract member to a type when there _are accessible_ (`public` or `protected`) constructors and the type is not `sealed`
 
 * Adding a constructor to a class which previously had no constructor, without also adding the default constructor
 
+* Introducing or removing an override to non-sealed types
```

Comment:
If we take it to the extreme every change can break someone somehow it is just whether or not we feel the change is risky enough. We generally have considered removing or adding an override as an acceptable change, which is why I would leave it in the allowed list and just update the comments about the risk.